bluepysnap/edges.py

```diff
@@ -114,6 +114,14 @@ def _dynamics_params_names(self):
     def _topology_property_names(self):
         return {Edge.SOURCE_NODE_ID, Edge.TARGET_NODE_ID}
 
+    @property
+    def population_config(self):
+        """Access the configuration for the population.
+
+        This configuration is extended the filepath to the h5 file containing the population.
```

Comment:
I do not understand this line of the comment :slightly_smiling_face:

Reply:
Me neither. I clarified it a good bit.